CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,39 @@
+# Code of Conduct
+
+## 1. Purpose
+
+The primary goal of this project is to foster an inclusive, respectful, and open community for everyone, regardless of their background or identity. This Code of Conduct outlines our expectations for participant behavior and the consequences for unacceptable behavior.
+
+## 2. Open Discussions & Respectful Feedback
+
+- **Encourage Diverse Ideas:** Everyone brings a unique perspective. Encourage different viewpoints, and listen openly to each other’s ideas.
```

Comment:
What if someone suggests using Windows Server instead of Unix? 😨 🤣

Reply:
I'm tolerant, BUT...

Reply:
This comment is mean and don't follow Code of conduct, and actually Windows is better then Unix, because 99% of games can be played on windows with easy, whereas not so many on Unix.

There are bigger problems. Like what if someone suggest to rewrite whole solution in PHP, or even worse in Ruby 🙈 

![troll](https://upload.wikimedia.org/wikipedia/en/9/9a/Trollface_non-free.png)